scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +64,14 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val):
+        """ convert float values to float type. keep string as is """
+        try:
+            return float(val)
```

Comment:
@simon3z the way I see it there are 3 ways to go:
1. What we are doing right now - for hawkular, we are deducing type from the metric itself, which can be both unsafe, and also limited: (for example: With the solution above, we can't send "1.3" as string metric)

2. A different solution would be to have the collector specifically state the metric type (with a default of gauge) and use that only for hawkular.

3. If we want these to be kept in one place, each collector could parse in runtime the vars template file that defines the type of this metric for zabbix. It's inefficient runtime-wise, but we will avoid hawk-zagg inconsistency.